### DISCUSSION TITLE
Adding additional dependency to linkerd helm chart

### DISCUSSION THREAD

**Discussion by shahriak** [linkerd/linkerd2#5179] [Q&A] [answered]
Hello All,

I am trying to add an additional dependency in requirements.yaml file. The goal is to deploy this chart along with all the other dependencies. When I try to build chart using `Helm dep build` command, I get "Error: directory ../partials not found" error. Does anybody know how to resolve this?

**Here's what the current requirement.yaml file looks like:**
- name: partials
  version: 0.1.0
  repository: file://../partials
- name: prometheus
  version: 0.1.0
  repository: file://../add-ons/prometheus
  condition: prometheus.enabled
- name: grafana
  version: 0.1.0
  repository: file://../add-ons/grafana
  condition: grafana.enabled
- name: tracing
  version: 0.1.0
  repository: file://../add-ons/tracing
  condition: tracing.enabled

**After my changes**
- name: my-app
  version: 0.1.0
  repository: file://../my-app
- name: partials
  version: 0.1.0
  repository: file://../partials
- name: prometheus
  version: 0.1.0
  repository: file://../add-ons/prometheus
  condition: prometheus.enabled
- name: grafana
  version: 0.1.0
  repository: file://../add-ons/grafana
  condition: grafana.enabled
- name: tracing
  version: 0.1.0
  repository: file://../add-ons/tracing
  condition: tracing.enabled

**Comment by grampelberg** [comment, marked as answer]
You need to have all those paths local. I would recommend creating your own chart and listing the Linkerd one as a dependency instead of doing it this way.

**Reply by shahriak** [reply]
Thank you for replying! Could you point me to the location from which I can add these components?

**Reply by grampelberg** [reply]
https://github.com/linkerd/linkerd2/tree/main/charts
